api/util/git_information_client.go

```diff
@@ -0,0 +1,206 @@
+package util
```

Comment:
Nit: wouldn't this fit better into a separate package like the log client which is in `api/log`?

Reply:
I moved quite some stuff into `api/validation` now.